llvm/include/llvm/CodeGen/SelectionDAGNodes.h

```diff
@@ -619,6 +619,9 @@ END_TWO_BYTE_PACK()
   /// Unique id per SDNode in the DAG.
   int NodeId = -1;
 
+  /// Index in worklist of DAGCombiner, or -1.
+  int CombinerWorklistIndex = -1;
```

Comment:
Does this increase the size of the node, or does it fill a hole (when pointers are 64-bit)?

Reply:
This fills 4 padding bytes, the size of an SDNode remains unchanged at 88 bytes (x86-64).